schema/vr.yaml

```diff
@@ -214,6 +206,32 @@ definitions:
             - $ref: "#/definitions/CURIE"
 
 
+  Haplotype:
+    additionalProperties: false
+    description: >-
+      A set of zero or more Alleles
```

Comment:
See comment above re: SHOULD/MUST and `0..*`/`1..*`. The current definition is `1..*` above, and `zero or more` here.

Reply:
Done.